ext/date/php_date.h

```diff
@@ -53,6 +53,7 @@ PHP_FUNCTION(getdate);
 PHP_METHOD(DateTime, __construct);
 PHP_METHOD(DateTime, __wakeup);
 PHP_METHOD(DateTime, __set_state);
+PHP_METHOD(DateTime, createFromImmutable);
 PHP_FUNCTION(date_create);
```

Comment:
I would say `fromImmutable`, since you aren't really creating it, but rather casting it